ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/forms/login/freemarker/models/LoginAttemptBean.java

```diff
@@ -0,0 +1,33 @@
+package edu.mit.keycloak.forms.login.freemarker.models;
+
+import org.keycloak.models.*;
+import org.keycloak.credential.CredentialModel;
+
+import java.util.stream.Stream;
+
+import edu.mit.keycloak.util.OpenAttribute;
+
+
+public class LoginAttemptBean {
```

Comment:
I would prepend "Ol" to this class name in order to more easily distinguish it against OOTB.